app/__init__.py

```diff
@@ -53,4 +74,14 @@ def remove_trailing_slash():
             else:
                 return redirect(request.path[:-1], code=301)
 
+    @application.before_request
+    def refresh_session():
+        session.permanent = True
+        session.modified = True
```

Comment:
No, nothing actually needs to be changed - but it will refresh the session timer.  At least I'm pretty sure that's what it does.